src/js/_enqueues/wp/webp-fallback.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * @output wp-includes/js/webp-fallback.js
+ */
+
+window.media = window.media || {};
+
+( function( document ) {
+	window.media.wpFallbackWebpImages = function( media ) {
```

Comment:
Similar to the file name comment, let's name this method more universally focused on image MIME types. Also see my above comment, we should put this into a `wp` global rather than `media`. Something like
```suggestion
	window.wp.imageMimeFallback = function( media ) {
```